packages/ui/components/Button/Button.ru.mdx

```diff
@@ -36,6 +45,41 @@ __Никогда__ не ставьте несколько кнопок `Приз
 </Button>
 ```
 
+## Цвета и размер
```

Comment:
отдельно секция цвета, отдельно секция размеры

для примера тебе текст, чтоб не писать магические посмотрите в `$UI.colors`, то обычно в доках наших можешь найти этот текст пишем вот так (possible values of ...)
Size is `m` by default. It can be changed by changing `size` prop. Possible values of property can be found in the `Sandbox` section at the bottom of the page.